animation_nodes/nodes/mesh/get_uv_map.py

```diff
@@ -0,0 +1,24 @@
+import bpy
+from ... base_types import AnimationNode
+
+class GetUVMapNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetUVMapNode"
+    bl_label = "Get UV Map"
+    errorHandlingType = "EXCEPTION"
+
+    def create(self):
+        self.newInput("Mesh", "Mesh", "mesh")
+        self.newInput("Text", "Name", "uvMapName")
+
+        self.newOutput("Vector 2D List", "Vectors", "vectors")
+
+    def execute(self, mesh, uvMapName):
+        if uvMapName == "":
+            self.raiseErrorMessage("UV map name can't be empty.")
+
+        uvMapCos = mesh.getUVMapPositions(uvMapName)
```

Comment:
`uvMapCos` -> `uvMapPositions`, here and else where.

Reply:
Okay.